docs/README.md

```diff
@@ -38,9 +38,10 @@ Integration tests | [![Build Status](https://dev.azure.com/datadoghq/dd-trace-do
     - Optional: ASP.NET and web development (to build samples)
   - Individual components
     - .NET Framework 4.7 targeting pack
-- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
```

Comment:
A few lines above, we should probably specify `Visual Studio 2019 (16.8)` (required for .NET 5.0)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1237/commits/8165b719b204f2681a7816737755960a4499e85b

Reply:
Thanks!